src/components/articles/Create.jsx

```diff
@@ -2,25 +2,32 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { Editor } from 'react-draft-wysiwyg';
 import { EditorState, convertToRaw } from 'draft-js';
+import TagsInput from 'react-tagsinput';
+import 'react-tagsinput/react-tagsinput.css'
 import {
   Form, Divider,
 } from 'semantic-ui-react';
 import { createArticle } from '../../actions/articlesActions';
 import 'react-draft-wysiwyg/dist/react-draft-wysiwyg.css';
 import './articles.scss';
 
-// This code is a modification of the sample login page from Semantic UI
 class createArticleView extends Component {
-  // Create a login view of the application
   constructor(props) {
     super(props);
     const state = this.props;
     this.state = {
       ...state,
+      tags: [],
```

Comment:
we are using redux to manage state. Kindly ensure that your states are from the global store.